lua/nvim-lsp-installer/platform.lua

```diff
@@ -25,6 +25,18 @@ M.is_unix = vim.fn.has "unix" == 1
 M.is_mac = vim.fn.has "mac" == 1
 M.is_linux = not M.is_mac and M.is_unix
 
+-- @return string @The libc found on the system, musl or glibc (glibc if ldd is not found)
+function M.get_libc()
+    local libc
+    local _, _, libc_exit_code = os.execute("ldd --version 2>&1 | grep -q musl")
+    if (libc_exit_code == 0) then
+        libc = "musl"
+    else
+        libc = "glibc"
```

Comment:
I guess these could be immediate returns instead of an assignment, since the variable is only used to hold the return value?

Reply:
100% could return right there. Imo -- probably better to return there immediately, I'll make that change here in a few minutes when I get a second.